pkg/cache/redis.go

```diff
@@ -35,19 +35,17 @@ func authKey(ctx context.Context) string {
 	return fmt.Sprintf("auth:%v,%v", identity.Identity.User.Username, identity.Identity.OrgID)
 }
 
+// repoConfigFileKey
+func pulpContentPathKey() string {
+	return "pulp-content-path"
+}
+
 // GetAccessList uses the request context to read user information, and then tries to retrieve the role AccessList from the cache
 func (c *redisCache) GetAccessList(ctx context.Context) (rbac.AccessList, error) {
 	accessList := rbac.AccessList{}
-	cmd := c.client.Get(ctx, authKey(ctx))
-	if errors.Is(cmd.Err(), redis.Nil) {
-		return nil, NotFound
-	} else if cmd.Err() != nil {
-		return nil, fmt.Errorf("redis error: %w", cmd.Err())
-	}
-
-	buf, err := cmd.Bytes()
+	buf, err := c.get(ctx, authKey(ctx))
 	if err != nil {
-		return nil, fmt.Errorf("redis bytes conversion error: %w", err)
+		return accessList, fmt.Errorf("redis get error")
```

Comment:
should this return the error retrieved from redis?

Reply:
yes, thanks